package.json

```diff
@@ -4,7 +4,7 @@
   "description": "The rendezvous service for libp2p-websocket-star",
   "main": "src/index.js",
   "bin": {
-    "rendezvous": "src/bin.js"
+    "websocket-star": "src/bin.js"
```

Comment:
Add both, so that it isn't a breaking change.

Reply:
Good point! Updated.